src/uvw/util.hpp

```diff
@@ -11,6 +11,12 @@
 #include <array>
 #include <uv.h>
 
+#ifdef _WIN32
```

Comment:
I suspect `R_CONSTEXPR` is useless, even if MSVC has not relaxed constexpr's contraints. See the comments below for further details.